src/config.rs

```diff
@@ -277,9 +278,26 @@ impl HookBuilder {
         HookBuilder {
             filters: vec![],
             capture_span_trace_by_default: false,
+            panic_note: None,
         }
     }
 
+    /// Add a custom note to the panic hook that will be printed
+    /// in the panic message.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// color_eyre::config::HookBuilder::default()
+    ///     .panic_note("consider reporting the bug at https://github.com/yaahc/color-eyre")
+    ///     .install()
+    ///     .unwrap()
+    /// ```
+    pub fn panic_note<S: Into<String>>(mut self, note: S) -> Self {
```

Comment:
for consistency with the helpers provided for `eyre` lets call this a `panic_section` instead of a `panic_note`

Later we might want to change it to be a `Vec` instead of an `Option` so users can provide multiple additional sections in the panic report, but for now I think as long as the API is consistent with an eventual direction we may go I think its fine.

```suggestion
    pub fn panic_section<S: Into<String>>(mut self, section: S) -> Self {
```